db/seeds.rb

```diff
@@ -43,6 +43,27 @@
                                school_type: "Academy",
                                gias_data: { 'ReligiousCharacter (name)': "None" })
 
+# A second school at the local authority to enable seeding a vacancy at multiple schools.
```

Comment:
Ignore this file's changes - they were reviewed in a separate PR which branched off this one (forgot that when I merged it!)